definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-general-referral-nonprod.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "LiveFrom": "08/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferral",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
```

Comment:
please use `CRUD`

It's going to be "re-triggerable" :) 

I don't know why, but when I didn't have `D` it didn't work.